app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -0,0 +1,57 @@
+require 'json'
+
+module Idv
+  module InPerson
+    class UspsLocationsController < ApplicationController
```

Comment:
Should we have feature flag condition(s) guarding this controller?

Reply:
can you explain a bit more?

Reply:
> can you explain a bit more?

Sure, my worry is that without a feature flag, anybody can make requests to this endpoint in production, even before we're ready to launch the feature. We use feature flags to make sure that this can't happen until we're ready to ship the feature.

And even though we wouldn't really expect those requests to come through normally because we're not showing the UI to users when the feature flag is disabled, typically it's still a good idea to feature flag the controllers to prevent someone who manually crafts a request from interacting with incomplete code.

Example:

https://github.com/18F/identity-idp/blob/6650d8edabbb0c6e8bbaf98665763a31ce62ee4e/app/controllers/idv/in_person_controller.rb#L3

https://github.com/18F/identity-idp/blob/6650d8edabbb0c6e8bbaf98665763a31ce62ee4e/app/controllers/idv/in_person_controller.rb#L17-L19

Reply:
addressed in #6757